File: FlexServ Backend/DotNet/AuthService/Dtos/ReviewResponse.cs
using System;

namespace AuthService.Dtos
{
    public class ReviewResponse
    {
        public long Id { get; set; }
        public long BookingId { get; set; }
        public string? CustomerName { get; set; }
        public string? ProviderName { get; set; }
        public int? Rating { get; set; }
        public string? Comment { get; set; }
        public DateTime? CreatedAt { get; set; }

        public ReviewResponse() { }

        public ReviewResponse(long id, long bookingId, string? customerName, string? providerName,
            int? rating, string? comment, DateTime? createdAt)
        {
            Id = id;
            BookingId = bookingId;
            CustomerName = customerName;
            ProviderName = providerName;
            Rating = rating;
            Comment = comment;
            CreatedAt = createdAt;
        }
    }
}

File: FlexServ Backend/SpringBoot/TransactionService/src/main/java/com/flexserv/dto/response/ReviewResponse.java
package com.flexserv.dto.response;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ReviewResponse {

    private Long id;
    private Long bookingId;
    private Long customerId;
    private String customerName;
    private Long providerId;
    private String providerCompanyName;
    private Integer rating;
    private String comment;
    private LocalDateTime createdAt;
}
